funcs/func_groupcount.c

```diff
@@ -51,37 +56,307 @@
+					Set(GROUP()=somethingelse)
+				On any Channel:
+					Assuming that channels still exist that have the above associated GROUP() assignents
+					Set(groups_found=${GROUP_LIST(.*foo.*)})
+				Given the above:
+					This will find the groups 'foobarbaz' and 'foobarbill' since they both match .*foo.* regex against the group name
+                                        Variable groups_found will be set to: foobarbaz,foobarbill
+			</example>
+			<example>
+				Find groups containing category name of 'bar'
+
+				On Channel 1:
+					Set(GROUP()=groupName@categoryName)
+				On Channel 2:
+					Set(GROUP()=foo@barbaz)
+				On Channel 3:
+					Set(GROUP()=foo@barbill)
+				On Channel 4:
+					Set(GROUP()=anything@somethingelse)
+				On any Channel:
+					Assuming that channels still exist that have the above associated GROUP() assignents
+					Set(groups_found=${GROUP_LIST(@.*bar.*)})
+
+				Given the above:
+					This will find the groups 'foo@barbaz' and 'foo@barbill' since they both match .*bar.* regex against the category name
+                                        Variable groups_found will be set to: foo@barbaz,foo@barbill
+			</example>
+			<example>
+				Find groups containing 'foo' and category name of 'bar'
+
+				On Channel 1:
+					Set(GROUP()=groupName@categoryName)
+				On Channel 2:
+					Set(GROUP()=foo@barbaz)
+				On Channel 3:
+					Set(GROUP()=foo@barbill)
+				On Channel 4:
+					Set(GROUP()=anything@somethingelse)
+				On any Channel:
+					Assuming that channels still exist that have the above associated GROUP() assignents
+					Set(groups_found=${GROUP_LIST(.*foo.*@.*bar.*)})
+
+				Given the above:
+					This will find the groups 'foo@barbaz' and 'foo@barbill' since they both match .*foo*. against the group name and .*bar.* regex against the category name
+                                        Variable groups_found will be set to: foo@barbaz,foo@barbill
+			</example>
+		</description>
+		<see-also>
+			<ref type="function">GROUP</ref>
+			<ref type="function">GROUP_CHANNEL_LIST</ref>
+		</see-also>
+	</function>
+	<function name="GROUP_CHANNEL_LIST" language="en_US">
+		<synopsis>
+			Find channels that are members of groups/categories by exact match
+		</synopsis>
+		<syntax argsep="@">
+			<parameter name="group"/>
+			<parameter name="category"/>
+		</syntax>
+		<description>
+			<para>Search for groups matching group@category and return the channels that are members of those groups, the group/category must match exactly</para>
+			<para>If only the group is specified, then return group@category results that only match against the group name</para>
+			<para>If only the categoryis specified, then return group@category results that only match the against the category name</para>
+			<para>If the search parameters are entirely empty, all channels having any kind of group membership will be returned.</para>
+			<example>
+				Find channels that are members of the group 'foobarbaz':
+
+				On Channel SIP/1234-0000001:
```

Comment:
Don't use the SIP technology for an example, using DAHDI, PJSIP, IAX2, or some core supported driver.

Reply:
Fixed